system/nagios-status-check/detectors-nagios-status-check.tf

```diff
@@ -0,0 +1,27 @@
+resource "signalfx_detector" "status_check" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Nagios check status"
+
+  program_text = <<-EOF
+        signal = data('nagios_state.state', filter=${module.filter-tags.filter_custom})${var.status_check_aggregation_function}.${var.status_check_transformation_function}(over='${var.status_check_transformation_window}').publish('signal')
+        detect(when(signal >= threshold(1)) and when(signal < 2)).publish('WARN')
+        detect(when(signal >= threshold(2))).publish('CRIT')
+  EOF
+
+  rule {
+    description           = "is not OK, please check the script output on the host"
```

Comment:
this is true but what about say explicitely it is a critical to respect the nagios terminology (even if you binded it to the same severity from signalfx)

Reply:
Yes way better, done.